lighthouse-core/lib/i18n/swap-flow-locale.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const swapLocale = require('./swap-locale.js');
+
+/**
+ * @param {LH.FlowResult} flowResult
+ * @param {LH.Locale} locale
+ */
+module.exports = function swapFlowLocale(flowResult, locale) {
+  const localizedFlowResult = JSON.parse(JSON.stringify(flowResult));
+  localizedFlowResult.steps = flowResult.steps.map(s => {
+    return {...s, lhr: swapLocale(s.lhr, locale).lhr};
```

Comment:
c'mon man, we're not even close to hitting line limits ;)

```suggestion
  localizedFlowResult.steps = flowResult.steps.map(step => {
    return {...step, lhr: swapLocale(step.lhr, locale).lhr};
```